config/jobs/kubernetes/sig-windows/windows-gce.yaml

```diff
@@ -166,7 +166,7 @@ periodics:
       - --cluster=
       - --extract=ci/k8s-master
       - --gcp-zone=us-west1-b
-      - --ginkgo-parallel=1
+      - --ginkgo-parallel=8
```

Comment:
I don't see anything excluding serial tests that can't run in parallel, I forget, will kubetest do that for you?